tools/webcam/camera.py

```diff
@@ -1,16 +1,19 @@
 import av
+import platform
 
 class Camera:
   def __init__(self, cam_type_state, stream_type, camera_id):
-    try:
-      camera_id = int(camera_id)
-    except ValueError: # allow strings, ex: /dev/video0
-      pass
     self.cam_type_state = cam_type_state
     self.stream_type = stream_type
     self.cur_frame_id = 0
 
-    self.container = av.open(camera_id)
+    print(f"opening {cam_type_state} at {camera_id}")
+
+    if platform.system() == "Darwin":
+      self.container = av.open(camera_id, format='avfoundation', container_options={"framerate": "30"})
```

Comment:
30 fps won't work... can we do 60 and throw out the other frames?

Reply:
I think we need a 60 fps webcam if we want 60 fps. Neither my built-in 720p webcam and neither my random bought 1080p external webcam doesn't support more than 30 fps. I'm still waiting for the NexiGo N60 but I don't think it supports 60 fps neither.